gcp-firebase/src/main/java/com/baeldung/gcp/firebase/auth/AccountAlreadyExistsException.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.gcp.firebase.auth;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class AccountAlreadyExistsException extends ResponseStatusException {
+
+    private static final long serialVersionUID = -8388485899470907402L;
```

Comment:
Is this necessary?

Reply:
Review incorporated.
It's out of habit to remove the IDE warning, but no it's not necessary. I've removed it.